config/config.go

```diff
@@ -31,6 +31,8 @@ const (
 	defaultTreeFileBufferSize       = 4096
 	defaultEstimatedLeavesPerSecond = 78000
 	defaultMaxRoundMembers          = 1 << 32
+	defaultSubmitFlushInterval      = 100 * time.Millisecond
+	defaultMaxSubmitBatchSize       = 1000
```

Comment:
the challenge is 32 bytes? so the queue size is ~32KB? have you tried with 10_000?

Reply:
Yes, the challenge is a 32B hash of a nipost challenge. On my machine, the 10k batch gives worse overall performance (`TestLoadSubmits` takes ~1.7 instead of 1.3s), but that should be tuned per machine.